cargo-process.el

```diff
@@ -255,15 +255,16 @@ Always set to nil if cargo-process--enable-rust-backtrace is nil"
              (not (member name cargo-process--no-manifest-commands)))
     (concat "--manifest-path " (cargo-process--project-root) "Cargo.toml")))
 
-(defun cargo-process--start (name command &optional last-cmd)
+(defun cargo-process--start (name command &optional last-cmd is-xdg)
```

Comment:
I still wish to use an argument, since not all commands (as far as I know it's only one) actually open external applications, so I wish to selectively adding `setsid` to those commands only.

Reply:
Fair point, could you make sure none of the other function break with this extra argument? Including the repeating of functions?